plugins/syslogng.yaml

```
@@ -0,0 +1,71 @@
+    default: "UTC"
+    relevant_if:
+      protocol:
+        equals: "rfc3164 (BSD)"
+
+# Set Defaults
+# {{$listen_address := default "0.0.0.0:514" .listen_address}}
+# {{$connection_type := default "udp" .connection_type}}
+# {{$protocol := default "rfc5424 (IETF)" .protocol}}
+# {{$location := default "UTC" .location}}
+
+# Pipeline Template
+pipeline:
+# {{ if eq $connection_type "udp" }}
+  - id: syslogng_input
+    type: udp_input
+    listen_address: {{ $listen_address }}
+    labels:
+      log_type: syslogng
+      plugin_id: {{ .id }}
+    output: syslogng_parser
+# {{ end }}
+
+# {{ if eq $connection_type "tcp" }}
+  - id: syslogng_input
```

Id's should be unique.
```suggestion
  - id: syslogng_tcp_input
```